otter/integration/lib/autoscale.py

```diff
@@ -181,20 +197,23 @@ def get_scaling_group_state(self, rcs):
             order.  If not found, the result code will be 404, and the state
             will be None.
         """
+        if success_codes is None:
```

Comment:
I generally like to check specifically for None as the sentinel value, in case someone wants to always fail the request by passing `success_codes=[]`, point well taken.  :)  I can definitely make this a one-liner, thank you!